graph/index.html

```diff
@@ -25,6 +25,7 @@
     var pageInfra = "";
     var pageStream = "";
     var pageArch = "";
+    var pageOci = false;
```

Comment:
Unused var?

Reply:
yeah, the whole navbar is broken. I tried to add a toggle for OCI but could not get it to work, that's a left over. Removed now